packages/core/template_project/app/src/main/AndroidManifest.xml

```diff
@@ -96,6 +100,10 @@
         <activity android:name="com.google.androidbrowserhelper.trusted.WebViewFallbackActivity"
             android:configChanges="orientation|screenSize" />
 
+        <activity android:name="com.google.androidbrowserhelper.trusted.PermissionRequestActivity"
```

Comment:
IIRC, this is part of an unreleased version of android-browser-helper, am I right? If that's the case, we also need to update the dependency on `app/build.gradle` when that version comes out.